app/domain/person/subscriptions.rb

```diff
@@ -21,6 +21,7 @@ def direct
 
   def exclusions
```

Comment:
does it make sense to cache this value since it's now used more than once in this class?